sharktank/sharktank/examples/export_paged_llm_v1.py

```diff
@@ -23,16 +24,35 @@ def main():
     parser = cli.create_parser()
     cli.add_input_dataset_options(parser)
     parser.add_argument(
-        "--output",
+        "--output_mlir",
         help="Output file path for exported MLIR file",
         default="/tmp/batch_llama_v1.mlir",
     )
+
+    parser.add_argument(
+        "--output_config",
+        help="Output file path for exported config file",
+        default="/tmp/batch_llama_v1.json",
+    )
     args = cli.parse(parser)
     dataset = cli.get_input_dataset(args)
 
     hp = configs.LlamaHParams.from_gguf_props(dataset.properties)
     model = PagedLlamaModelV1(dataset.root_theta, LlamaModelConfig(hp))
 
+
+    def generate_params_json(hp, prefill_bs: list[int], decode_bs: list[int]):
```

Comment:
https://github.com/nod-ai/sharktank/actions/runs/9036618518/job/24833851082?pr=24#step:4:118
```suggestion
    model = PagedLlamaModelV1(dataset.root_theta, LlamaModelConfig(hp))

    def generate_params_json(hp, prefill_bs: list[int], decode_bs: list[int]):
```

lint wants one less newline here